test/test_integration.py

```diff
@@ -200,12 +200,25 @@
             'expect_out': 'b,c\n2,3\n5,6\n'}
 
```

Comment:
The only change in this file is the addition of the extra_newline values,
The rest is code style.